### PR TITLE
style(frontend): adjust position of button in Send modal

### PR DESCRIPTION
**Pull request by AntonioVentilii-DFINITY**
# Motivation

Spotted by @artkorotkikh-dfinity , the button in the first step of the Send modal is in the wrong position. This PR aims to solve it mirroring what is being done in the other steps.

### Before

Screenshot 2024-09-13 at 22 18 20

## After

Screenshot 2024-09-13 at 22 17 56

# To-Do

- [ ] Create a generic component to wrap the content of the wizard modals with a slot for the buttons.

### CODE REVIEW

File: src/frontend/src/lib/components/send/SendTokensList.svelte [hunk 1]
```diff
@@ -21,18 +22,24 @@
...
+				{$i18n.tokens.manage.text.all_tokens_zero_balance}
+			</p>
+		{/if}
+	</TokensSkeletons>
+</div>
 
-<button class="secondary full center text-center" on:click={modalStore.close}>
-	{$i18n.core.text.close}
-</button>
+<ButtonGroup>
```
Comment (peterpeterparker) [inline src/frontend/src/lib/components/send/SendTokensList.svelte:41]: The `ButtonGroup` is not need I think given there is only a single button. Can you try?

Comment (AntonioVentilii-DFINITY) [reply src/frontend/src/lib/components/send/SendTokensList.svelte:41]: nono, you're correct, not needed

File: src/frontend/src/lib/components/send/SendTokensList.svelte [hunk 2]
```diff
@@ -21,18 +21,22 @@
...
+		{#if tokens.length === 0}
+			<p class="mt-4 mb-6 text-dark opacity-50">
+				{$i18n.tokens.manage.text.all_tokens_zero_balance}
+			</p>
+		{/if}
+	</TokensSkeletons>
+</div>
 
-<button class="secondary full center text-center" on:click={modalStore.close}>
+<button class="secondary block flex-1 mb-2" on:click={modalStore.close}>
```
Comment (peterpeterparker) [inline src/frontend/src/lib/components/send/SendTokensList.svelte:40]: Now that `ButtonGroup` has been removed, you ned to change the style class?

`block flex-1 mb-2` is needed?

Comment (AntonioVentilii-DFINITY) [reply src/frontend/src/lib/components/send/SendTokensList.svelte]: good catch! I forgot to revert to the original classes

File: src/frontend/src/lib/components/send/SendTokensList.svelte [hunk 3]
```diff
@@ -21,18 +21,22 @@
...
+		{#if tokens.length === 0}
+			<p class="mt-4 mb-6 text-dark opacity-50">
+				{$i18n.tokens.manage.text.all_tokens_zero_balance}
+			</p>
+		{/if}
+	</TokensSkeletons>
+</div>
 
-<button class="secondary full center text-center" on:click={modalStore.close}>
+<button class="secondary full center text-center mb-2" on:click={modalStore.close}>
```
Comment (peterpeterparker) [inline src/frontend/src/lib/components/send/SendTokensList.svelte:40]: `mb-2` is expected?

**Review by peterpeterparker — APPROVED** [review]
LGTM, thx

File: src/frontend/src/lib/components/send/SendTokensList.svelte (hunk 3 above)
Comment (AntonioVentilii-DFINITY) [reply src/frontend/src/lib/components/send/SendTokensList.svelte:40]: yessir, to have the same margin as the others

**Pull request merged**